public/app/plugins/datasource/grafana/datasource.ts

```diff
@@ -85,13 +85,6 @@ export class GrafanaDatasource extends DataSourceWithBackend<GrafanaQuery> {
         let channel = templateSrv.replace(target.channel, request.scopedVars);
         const { filter } = target;
 
-        // Help migrate pre-release channel paths saved in dashboards
-        // NOTE: this should be removed before V8 is released
-        if (channel && channel.startsWith('telegraf/')) {
```

Comment:
should likely be in a different PR.. but I stumbled into it here, and did not want to lose it!